tests/policies/candidate_selectors/vllm/test_vllm_resource_fit_selector.py

```diff
@@ -288,7 +291,7 @@ def workers():
 
 
 @pytest.mark.asyncio
-async def test_auto_schedule_to_2_worker_2_gpu(config):
+async def test_manual_schedule_to_2_worker_2_gpu_with_gpus_per_replica_1(config):
```

Comment:
Can we leverage `@pytest.mark.parametrize` to group up the test input, including `test_manual_schedule_to_2_worker_2_gpu_with_gpus_per_replica_1` and `test_manual_schedule_to_2_worker_4_gpu_with_gpus_per_replica_2`?

It seems we are committed to continually expanding the range of function names to reflect the test goals. However, as the amount of orthogonal content increases, maintaining this will become more challenging.